pyogrio/tests/test_core.py

```diff
@@ -460,15 +460,19 @@ def test_read_info_force_feature_count(data_dir, layer, force, expected):
     assert meta["features"] == expected
 
 
-@pytest.mark.parametrize(
-    "force_total_bounds, expected_total_bounds",
-    [(True, (-180.0, -90.0, 180.0, 83.64513)), (False, None)],
-)
-def test_read_info_force_total_bounds(
-    tmpdir, naturalearth_lowres, force_total_bounds, expected_total_bounds
-):
-    # Geojson files don't hava a fast way to determine total_bounds
+@pytest.mark.parametrize("force_total_bounds", [True, False])
+def test_read_info_force_total_bounds(tmpdir, naturalearth_lowres, force_total_bounds):
+    # Geojson files don't have a fast way to determine total_bounds for GDAL <3.9.0
```

Comment:
Ideally for later we should find a new file format to test with that doesn't support fast total bounds?

Reply:
It looks like GeoJSONSeq still doesn't support fast total bounds, so I reverted back to using that directly.